resources/knative-eventing/charts/knative-eventing/uninstall.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -eux -o pipefail
+echo "starting deletion of knative-eventing core custom resources"
+resources=(
+  "apiserversources.sources.eventing.knative.dev"
+  "containersources.sources.eventing.knative.dev"
+  "cronjobsources.sources.eventing.knative.dev"
+  "sinkbindings.sources.eventing.knative.dev"
+
+  "eventtypes.eventing.knative.dev"
+  "brokers.eventing.knative.dev"
+  "triggers.eventing.knative.dev"
+
+  "channels.messaging.knative.dev"
+  "parallels.messaging.knative.dev"
+  "sequences.messaging.knative.dev"
+  "subscriptions.messaging.knative.dev"
+)
+
+for resource in "${resources[@]}"; do
+  echo "starting deletion of ${resource} custom resources"
+  if kubectl get crd "${resource}"; then
+    kubectl \
+      delete \
+      --ignore-not-found \
+      "${resource}" \
+      --all-namespaces \
+      --all
+  else
+    echo "CRD ${resource} unknown to apiserver"
+  fi
+  echo "finished deletion of ${resource} custom resources"
+done
+
+echo "done"
```

Comment:
Please add a newline

Reply:
I really would like to have a style guideline for that and also tooling around it. I personally don't care whether there is a newline or not. I would like to not have to wonder about it. I will leave it as it is until we have something like this :)